src/Components/SigDetailsTable/NewSigDetailsTable.js

```diff
@@ -84,15 +85,17 @@ const NewSigDetailsTable = ({ ruleName, affectedCount, isEmptyAccount }) => {
     useState(false);
   const [sigDetailsTableGroups, setSigDetailsTableGroups] = useState({});
   const [expandedRows, setExpandedRows] = useState([]);
+  const [searchParams, setSearchParams] = useSearchParams();
 
   const initialState = {
     tableVars: {
       limit: 10,
       offset: 0,
       orderBy: 'LAST_SCAN_DATE_DESC',
-      displayName: '',
-      hostGroupFilter: undefined,
-      sigMatchStatusFilter: undefined,
+      displayName: searchParams.get('displayName') || '',
+      hostGroupFilter:
+        searchParams.get('hostWorkspace')?.split(',') || undefined,
```

Comment:
I believe we do not have to have else condition with **undefined** value. Because, **searchParams.get('hostWorkspace')?.split(',')** already results in undefined if `hostWorkspace` does not exist. The same applies below line of code
```suggestion
        searchParams.get('hostWorkspace')?.split(','),
```